core/capstanignore.go

```diff
@@ -22,21 +22,21 @@ type Capstanignore interface {
 	IsIgnored(path string) bool
 }
 
-var CAPSTANIGNORE_ALWAYS []string = []string{"/meta/*", "/mpm-pkg/*", "/.git/*"}
+var CAPSTANIGNORE_ALWAYS []string = []string{"/meta", "/mpm-pkg", "/.git"}
 
-// CapstanignoreInit creates a new capstanignore struct that is
+// CapstanignoreInit creates a new Capstanignore struct that is
 // used when deciding whether a file should be included in unikernel
 // or not. You can provide `path` to the .capstanignore file to load
 // it or pass empty string "" if you have none. Note that once having
 // capstanignore struct you can load as many files as you want (using
 // .LoadFile function) or manually add as many patterns as you like
 // (using .AddPattern function).
-func CapstanignoreInit(path string, verbose bool) Capstanignore {
-	c := capstanignore{verbose: verbose}
+func CapstanignoreInit(path string) Capstanignore {
+	c := capstanignore{}
 
 	// Load capstanignore file if path was given.
 	if path != "" {
-		if err := c.LoadFile(path); verbose && err != nil {
+		if err := c.LoadFile(path); err != nil {
```

Comment:
Thanks for this, I've propagated the error. I think that accessing pattern cannot result in error since we perform no validation, but in case `.capstanignore` has improper permission it's better to shout this out loud. Done.